ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -87,17 +87,12 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
     const { id: scope_id } = this.modelFor('scopes.scope');
     await this.getAllTargets(transition, scope_id);
     const projects = this.modelFor('scopes.scope.projects');
-    const projectIds = projects.map((project) => project.id);
+    const orgScope = this.modelFor('scopes.scope');
 
     const filters = { scope_id: [], id: { values: [] }, type: [] };
     scopes.forEach((scope) => {
       filters.scope_id.push({ equals: scope });
     });
-    if (scopes.length === 0) {
-      projectIds.forEach((projectId) => {
-        filters.scope_id.push({ equals: projectId });
```

Comment:
Depends. I actually added this because if you were to filter by scope at the global level followed changing the org scope via the header the filters will clear out. Then if you were to change the scope back to global via the header, the filters you did have would still be there. I only added scope to the the `resetController` but this will be true for any queryParam on the route. We should either clear all filters or just remove the `resetController` entirely. Happy to chat more about this.

Reply:
Yeah I agree with that, I did find it weird that some filters weren't cleared out. I don't particularly have a preference to either clearing all filters or none, but I do agree that it should be one or the other